d2renderers/d2svg/d2svg.go

```diff
@@ -1046,9 +1165,9 @@ func RenderText(text string, x, height float64) string {
 	return strings.Join(rendered, "")
 }
 
-func embedFonts(buf *bytes.Buffer, fontFamily *d2fonts.FontFamily) {
+func embedFonts(buf *bytes.Buffer, fontFamily *d2fonts.FontFamily) string {
```

Comment:
why this change?

Reply:
not sure rn, also I think I'll write some kind of function like `addOnTrigger` to generalize the font embedding and possibly others.